src/templates/next/layouts/Collection/CollectionClient.tsx

```diff
@@ -170,6 +196,41 @@ const getPaginatedItems = (
   )
 }
 
+const updateAppliedFilters = (
+  appliedFilters: AppliedFilter[],
+  setAppliedFilters: React.Dispatch<React.SetStateAction<AppliedFilter[]>>,
+  filterId: string,
```

Comment:
what about 
`setAppliedFilters: (AppliedFilter: AppliedFilter[]) => void,` for less verbose code?

Reply:
Updated in b9132a97c2c23386bb81a2a7736c9e2f658d9565!